yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/MultiExecutor.java

```diff
@@ -0,0 +1,124 @@
+package tech.ytsaurus.client;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import javax.annotation.Nullable;
+
+public class MultiExecutor implements Closeable {
```

Comment:
To make the pr easier to review, I suggest returning all classes to their previous places (so that the diff is minimal).